.github/workflows/validate_pr_title.yml

```
@@ -9,7 +9,7 @@ on:
 
 jobs:
   main:
```

```suggestion
  validate-pr-title:
```

We can change this and remove the name as the default is the key